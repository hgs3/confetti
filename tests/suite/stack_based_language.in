push 1
push 2
add     # Pop the top two numbers and push their sum.
pop $x  # Pop the sum and store it in $x.
print "1 + 2 ="
print $x
