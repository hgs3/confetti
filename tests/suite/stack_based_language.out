<push> <1>
<push> <2>
<add>
<pop> <$x>
<print> <1 + 2 =>
<print> <$x>
